src/providers/moduleCalls.ts

```diff
@@ -131,13 +131,16 @@ export class ModuleCallsDataProvider implements vscode.TreeDataProvider<ModuleCa
 
     const editor = activeEditor.document.uri;
     const documentURI = Utils.dirname(editor);
-    const handler = this.handler.getClient();
-    if (handler === undefined) {
+    if (this.handler === undefined) {
       return [];
     }
-    await handler.onReady();
+    await this.handler.onReady();
 
-    const commandSupported = this.handler.clientSupportsCommand(`terraform-ls.module.calls`);
+    // clientSupportsCommand(`terraform-ls.module.calls`)
+    const commandSupported =
+      this.handler.initializeResult?.capabilities.executeCommandProvider?.commands.includes(
+        'terraform-ls.module.calls',
+      );
```

Comment:
We lose the neat `clientSupportsCommand` function by removing ClientHandler so we have to use this long property accessor here to check if the command is supported.

Since we ship the LS with the extension now and the versions are in lockstep, we don't technically need to check anymore if it's supported, but I'm leaving this for now as it does not harm anything to check here.

Reply:
Maybe we can add a small function to `clientHelpers.ts` and keep the check just to be safe?

```ts
export function clientSupportsCommand(initializeResult: InitializeResult | undefined, cmdName: string): boolean {
  if (!initializeResult) {
    return false;
  }

  return initializeResult.capabilities.executeCommandProvider?.commands.includes(cmdName) ?? false;
}
```

And call it everywhere, where we check for commands:

```ts
const commandSupported = clientSupportsCommand(this.client.initializeResult, 'terraform-ls.module.calls');
```

Reply:
Done!